src/music/sound.py

```diff
@@ -1,21 +1,46 @@
+SYMBOLS = ['C', 'C#', 'D', 'D#', 'E',
+           'F', 'F#', 'G', 'G#', 'A', 'A#', 'B']
+REVERSE_SYMBOLS = {
+    s: i for i, s in enumerate(SYMBOLS)
+}
+REVERSE_SYMBOLS.update(
+    {
+        s.lower(): i for i, s in enumerate(SYMBOLS)
+    }
+)
+REVERSE_SYMBOLS.update(
+    {
+        'cis': 1,
+        'dis': 3,
+        'fis': 6,
+        'gis': 8,
+        'ais': 10
+    }
+)
+
+
 class Sound:
-    def __init__(self, note, timestamp=None, duration=None):
+    def __init__(self, note=None, timestamp=None, duration=None,
+                 duration_signature=None, symbol=None):
         self.note = note
+        if symbol is not None:
+            self.note = REVERSE_SYMBOLS[symbol]
         self.timestamp = timestamp
         self.duration = duration
```

Comment:
Change to `duration_ms` to match documentation

Reply:
Done